src/api-to-friendly.ts

```diff
@@ -24,10 +24,22 @@ const apiToFriendlyRelation = (
   relationDefinition: Userset,
   relations: string[],
   idx: number,
+  metadata: Userset | Metadata | undefined,
   newSyntax: string[],
 ) => {
   const relationKeys = Object.keys(relationDefinition);
-  const define = [`    ${Keywords.DEFINE} ${relation}${relationKeys?.length ? ` ${Keywords.AS} ` : ""}`];
+  const relationMetadata = (metadata as Metadata)?.relations?.[relation];
+  
+  let allowedTypesArray = relationMetadata?.directly_related_user_types?.map((relationReference) => {
+    if (relationReference.relation) {
+      return `${relationReference.type}#${relationReference.relation}`;
+    }
+    return relationReference.type;
+  }) || [];
+  
+  let allowedTypes = allowedTypesArray.length > 0 ? `: [${allowedTypesArray.join(",")}]` : "";
```

Comment:
```suggestion
  let allowedTypes = allowedTypesArray.length ? `: [${allowedTypesArray.join(",")}]` : "";
```

Reply:
similar to above, use const if we are not changing.